### PR TITLE
Chronos: Add simulator to Chronos API

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

**Comment by liangs6212** [comment]
API doc: https://liangs.readthedocs.io/en/doc_simaltor/doc/PythonAPI/Chronos/simulator.html

File: pyzoo/zoo/chronos/simulator/doppelganger_simulator.py [hunk 1]
```diff
@@ -14,20 +14,17 @@
 # limitations under the License.
 #
 
-import numpy as np
 import pickle
 import os
+import numpy as np
+import torch
```
Comment (TheaperDeng) [inline pyzoo/zoo/chronos/simulator/doppelganger_simulator.py:20]: seems that we have got a torch import on line 26

**Review by TheaperDeng — COMMENTED** [review]
others lgtm

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/ZOO-PR-Validation/6473/
http://10.112.231.51:18888/job/ZOO-PR-Scala-Spark-2.4/4894/
http://10.112.231.51:18888/job/ZOO-PR-Python-Spark-3.1-py37-ray/1944/

**Pull request merged**